plugin/db/util/driverutil.go

```diff
@@ -16,6 +17,30 @@ import (
 	"go.uber.org/zap"
 )
 
+// CountWriter counts the written bytes to the embedded io.Writer.
+// CountWriter itself is also a io.Writer, which makes it a drop-in replacement where an io.Writer is requested.
+type CountWriter struct {
+	w          io.Writer
+	bytesCount int64
+}
+
+// NewCountWriter returns a new instance of CountWriter.
+func NewCountWriter(w io.Writer) *CountWriter {
+	return &CountWriter{w: w}
+}
+
+// Write implements the io.Writer interface
+func (w *CountWriter) Write(p []byte) (int, error) {
```

Comment:
Instead of keeping track of size here, can we get the dump file size after we finish the writes?

Reply:
SG.